packages/assets-controllers/src/TokenRatesController.ts

```diff
@@ -1,13 +1,17 @@
 import type { BaseConfig, BaseState } from '@metamask/base-controller';
-import { BaseController } from '@metamask/base-controller';
 import {
   safelyExecute,
   handleFetch,
   toChecksumHexAddress,
   FALL_BACK_VS_CURRENCY,
   toHex,
 } from '@metamask/controller-utils';
-import type { NetworkState } from '@metamask/network-controller';
+import type {
+  NetworkClientId,
+  NetworkController,
```

Comment:
see https://github.com/MetaMask/core/pull/2015/files#r1393439534

```suggestion
  NetworkControllerGetNetworkClientByIdAction,
```